src/actix/api/collections_api.rs

```diff
@@ -151,6 +158,7 @@ async fn update_collection_cluster(
 // Configure services
 pub fn config_collections_api(cfg: &mut web::ServiceConfig) {
     cfg.service(get_collections)
+        .service(get_aliases) // higher priority than /collections/{name}
```

Comment:
There is a subtle issue here as the get_aliases needs to have a higher priority than get_collection.
GET collections/aliases vs GET collections/{name}

Otherwise the API is not visible.

Maybe we should forbid the creation of collections name `aliases`?

Reply:
WDYT about `GET /aliases` for all aliases and `GET /collections/<name>/aliases` for aliases associated with specified colelction?

Similar to `/snapshots`

Reply:
Thanks for the suggestion :+1: 
Done in https://github.com/qdrant/qdrant/pull/1363/commits/4b6ef7875d84c0e8feccff0e31805b80aa1d58ca